input/community/index.md

```diff
@@ -2,6 +2,6 @@ Title: Links to the Axiom3D Communities
 ---
 
 - [Join the Slack Community!](join-slack)
+<div><img style="width: 20%;" src="/assets/img/slacklogo.png" alt=""/></div>
```

Comment:
1) Use markdown
2) Resize the image if it's needed.

Reply:
Ok there are resizing codes but it seems only for various platforms. I’ll try some.

Reply:
This has been resolved with new code from source.